LibraBFT/Impl/Properties/VotesOnce.agda

```diff
@@ -74,9 +75,8 @@ module LibraBFT.Impl.Properties.VotesOnce where
 
   isValidNewPart⇒fSE {pk}{v'}{pre}{theStep = step-peer {β} {postst} {outs} {.pre} pstep} hpk (r , ¬init , ¬sentb4 , mws , refl , zefl , vpk)
      | inj₁ thisStep
-     | step-honest {.β} hstep
-     with Any-satisfied-∈ (Any-map⁻ thisStep)
-  ...| nm , refl , nm∈outs
+     | step-honest {.β} hstep with senderMsgPair∈⇒send∈ outs thisStep
```

Comment:
We generally prefer to put the with on its own line, aligned with the `|`.  Feel free to push back, but if you don't have a preference, please stick with our convention.